File: src/main/java/com/linebot/model/qiita/Stocker.java
package com.linebot.model.qiita;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Stocker {
    // https://qiita.com/api/v2/docs#get-apiv2itemsitem_idstockers

    private String description;
    private String id;
    private String name;
    private String organization;
    @JsonProperty("profile_image_url")
    private String profileImageUrl;
    @JsonProperty("website_url")
    private String websiteUrl;
}

File: src/main/kotlin/com/linebot/model/qiita/Stocker.kt
package com.linebot.model.qiita

import com.fasterxml.jackson.annotation.JsonProperty

data class Stocker(
        // https://qiita.com/api/v2/docs#get-apiv2itemsitem_idstockers

        var description: String? = null,
        var id: String? = null,
        var name: String? = null,
        var organization: String? = null,
        @JsonProperty("profile_image_url")
        var profileImageUrl: String? = null,
        @JsonProperty("website_url")
        var websiteUrl: String? = null
)
